datafusion/sqllogictest/test_files/timestamps.slt

```diff
@@ -3249,7 +3249,7 @@ statement error
 select to_local_time('2024-04-01T00:00:20Z'::timestamp, 'some string');
 
 # invalid argument data type
-statement error The to_local_time function can only accept Timestamp as the arg got Utf8
+statement error DataFusion error: Error during planning: Internal error: Expect TypeSignatureClass::Timestamp but received NativeType::String, DataType: Utf8
```

Comment:
this seems wrong to me that a user error on query returns an internal error  -- it should be a planning error I would think

Reply:
I intend to look into cleaning up the error messages from using the coercion signature api soon 👍